src/libs/state.ts

```diff
@@ -30,6 +39,8 @@ export const authStore = atom<any>({
 
 export const getUser = () => authStore.get().user;
 
+export const getSessionId = () => (authStore.get().sessionId === undefined ? 'undefined' : authStore.get().sessionId);
```

Comment:
Since this is only used for formatting values sent to MixPanel, let's move it to Metrics. With it here, other code could end up using it, which would reduce type safety since it replaces `undefined` with `"undefined"`.

Reply:
I agree that this should be a metrics only fn. The issue we are trying to solve is outlined in the thread here: https://broadinstitute.slack.com/archives/C01EHNUM73R/p1694542792211539.